## Papers to read

### Protein GYM


-Clinpred: A prediction tool to identify disease-relevant nonsynonymous single-nucleotide variants.
- Unified rational protein engineering with sequence-based deep representation learning
- A fundamental protein property, thermodynamic stability, revealed solely from large-scale measurements of
protein function. 
- The mutational landscape of a prion-like domain. Nature Communications
- Genome-wide prediction of disease variant effects with a deep protein language model
- Identifying mendelian disease genes with the variant effect scoring tool.
- Profiling of the drug resistance of thousands of src tyrosine kinase mutants uncovers a regulatory network that couples autoinhibition to catalyticdomain dynamics. 
- Correlation of fitness landscapes from three orthologous TIM barrels originates from sequence and structure constraints.
- HotProtein: A Novel Framework for Protein Thermostability Prediction and Editing
- Predicting the functional effect of amino acid substitutions and indels
- LIP: Benchmark tasks in fitness landscape inference for proteins
- Robust deep learning-based protein sequence design using ProteinMPNN
- Deep Sequencing of Systematic Combinatorial Libraries Reveals beta-Lactamase Sequence Constraints at High Resolution
- Towards cracking the language of lifes code through self-supervised deep learning and high performance computing.
- Joint evolutionary trees: a large-scale method to predict protein interfaces based on sequence sampling.
- Saturation variant interpretation using CRISPR prime editing. 
- Mapping the energetic and allosteric landscapes of protein binding domains.
- Comprehensive fitness landscape of SARS-CoV-2 M pro reveals insights into viral resistance mechanisms
- Disease variant prediction with deep generative models of evolutionary data.
- Fitness Effects of Single Amino Acid Insertions and Deletions in TEM-1 beta-Lactamase.
- Experimental illumination of a fitness landscape.
- Axial attention in multidimensional transformers.
- SoluProt: prediction of soluble protein expression in escherichia coli.
- Mutation effects predicted from sequence co-variation.
- REVEL: an ensemble method for predicting the pathogenicity of rare missense variants.

- Cross-protein transfer learning substantially improves zero-shot prediction of disease variant effects.
-  Going Beyond Individual Input-Output Pairs in Deep Learning
- Evolutionary-scale prediction of atomic-level protein structure with a language model.
- Language models enable zero-shot prediction of the effects of mutations on protein function.
- Tranception: protein fitness prediction with autoregressive transformers and inference-time
retrieva
- Proteinnpt: Improving protein property prediction and design with non-parametric transformers
- Navigating the protein fitness landscape with gaussian processes.
- Predicting the functional, molecular, and phenotypic consequences of amino acid substitutions using hidden markov models
- 

### ZymCTRL



- Computational design of enone-binding proteins with catalytic activity for the
morita–baylis–hillman reaction 
- Robust deep learning–based protein sequence design using ProteinMPNN
- From sequence to function through structure: deep learning for protein design
- De novo computational design of retro-aldol enzymes.
- Strategies for designing non-natural enzymes and binder
- Evaluating protein transfer learning with TAPE
- MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets




























